project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.6")
 
-addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.7")
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.18")
```

Comment:
When using the old version TeamCity wasn't picking up the build number correctly; this upgrade fixes that problem.